src/AssemblerHttpServer.cpp

```diff
@@ -55,35 +55,7 @@ void Assembler::processRequest(
 {
     // Process a documentation request.
     const string& keyword = request.front();
-    if(keyword.size()>6 && keyword.substr(0, 6)=="/docs/") {
-
-        // Extract the file name.
-        const string name = keyword.substr(6);
-
-        // If it contains "/", reject it.
-        if(name.find('/') != string::npos) {
-            writeHtmlBegin(html);
-            html << "Unknown documentation file " << name;
-            writeHtmlEnd(html);
-            return;
-        }
-
-        // Construct the full file name and open it.
-        const string fileName = httpServerData.docsDirectory + "/" + name;
-        ifstream file(fileName);
-        if(!file) {
-            writeHtmlBegin(html);
-            html << "Could not open " << fileName;
-            writeHtmlEnd(html);
-        }
-
-        // Send it to html.
-        html << "\r\n" << file.rdbuf();
-        return;
-    }
-
-
-
+    
```

Comment:
Perhaps leave this in under `#ifdef` or commented out, to make it easier to reinstate if we decide to do that in the future?